sdk/python/lib/pulumi/runtime/resource.py

```diff
@@ -197,22 +197,23 @@ def do_resolve(value: Any, perform_apply: bool, exn: Optional[Exception]):
             )
 
             def do_rpc_call():
-                if monitor:
-                    # If there is a monitor available, make the true RPC request to the engine.
-                    try:
-                        return monitor.ReadResource(req)
-                    except grpc.RpcError as exn:
-                        # See the comment on invoke for the justification for disabling
-                        # this warning
-                        # pylint: disable=no-member
-                        if exn.code() == grpc.StatusCode.UNAVAILABLE:
-                            sys.exit(0)
-
-                        details = exn.details()
-                    raise Exception(details)
-                else:
+                if monitor is None:
                     # If no monitor is available, we'll need to fake up a response, for testing.
-                    return RegisterResponse(create_test_urn(ty, name), None, resolver.serialized_props)
```

Comment:
it feels like this could be reverted. did any actual functional change happenhere?

Reply:
Same as elsewhere - this is forced by new version of `pylint`.